CONTRIBUTING/writing-style.md

```diff
@@ -0,0 +1,42 @@
+# Writing style
```

Comment:
There is no reference to this file from any of the other files, probably will be helpful to link from the contributing README.md

Reply:
fixed